applications/StructuralMechanicsApplication/tests/membrane_test/Membrane_hemisphere_test_parameters.json

```diff
@@ -0,0 +1,82 @@
+{
+    "problem_data"     : {
+        "problem_name"  : "hemissphere",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 0,
+        "start_time"    : 0.0,
+        "end_time"      : 1.0
+    },
+    "solver_settings"  : {
+        "solver_type"                        : "Static",
+        "model_part_name"                    : "Structure",
+        "domain_size"                        : 3,
+        "echo_level"                         : 0,
+        "analysis_type"                      : "non_linear",
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "membrane_test/Membrane_hemisphere_test"
+        },
+        "material_import_settings"           : {
+            "materials_filename" : "membrane_test/Membrane_hemisphere_test_material.json"
+        },
+        "time_stepping"                      : {
+            "time_step" : 1.1
+        },
+        "line_search"                        : false,
+        "convergence_criterion"              : "residual_criterion",
+        "displacement_relative_tolerance"    : 1e-10,
+        "displacement_absolute_tolerance"    : 1e-10,
+        "residual_relative_tolerance"        : 1e-10,
+        "residual_absolute_tolerance"        : 1e-10,
+        "max_iteration"                      : 50,
+        "problem_domain_sub_model_part_list" : ["Parts_structure"],
```

Comment:
`problem_domain_sub_model_part_list` and `processes_sub_model_part_list` are deprecated, please remove them